### ISSUE TITLE
Step Functions graph won't render without default variable on choice state

### ISSUE THREAD

**Issue by JoshRacine** [aws/aws-toolkit-vscode#1030]
**Describe the bug**
Failure to render step functions graphs when the definition has a choice state without a default next state. Per the [docs](https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-choice-state.html) default is optional, but recommended.

`[Error - 4:50:36 PM] Error while validating file:///home/josh/projects/project1/stepfunctions.asl.json: Cannot read property 'valueNode' of undefined
TypeError: Cannot read property 'valueNode' of undefined
    at /home/josh/.vscode-server/extensions/amazonwebservices.aws-toolkit-vscode-1.8.0/dist/src/stepFunctions/asl/aslServer.js:2:96875
    at Array.forEach (<anonymous>)
    at e (/home/josh/.vscode-server/extensions/amazonwebservices.aws-toolkit-vscode-1.8.0/dist/src/stepFunctions/asl/aslServer.js:2:95717)
    at /home/josh/.vscode-server/extensions/amazonwebservices.aws-toolkit-vscode-1.8.0/dist/src/stepFunctions/asl/aslServer.js:2:96320
    at Array.forEach (<anonymous>)
    at Object.e [as default] (/home/josh/.vscode-server/extensions/amazonwebservices.aws-toolkit-vscode-1.8.0/dist/src/stepFunctions/asl/aslServer.js:2:95717)
    at Object.<anonymous> (/home/josh/.vscode-server/extensions/amazonwebservices.aws-toolkit-vscode-1.8.0/dist/src/stepFunctions/asl/aslServer.js:2:58059)
    at Generator.next (<anonymous>)
    at s (/home/josh/.vscode-server/extensions/amazonwebservices.aws-toolkit-vscode-1.8.0/dist/src/stepFunctions/asl/aslServer.js:2:57086)`

**Steps to reproduce:**
Attempt to render any state machine with a choice state without a default value

**Expected behavior**
Graph should render regardless of default option

**Desktop:**

-   OS: Win 10 using Ubuntu 18.04 WSL
-   Visual Studio Code Version: 1.44.1
-   AWS Toolkit Version: 1.8.0

**Comment by wong-a** [comment]
Confirming this is a bug. It can be reproduced with the **Catch failure** starter template selected from the `AWS: Create a new Step Functions state machine` command.

**Comment by wong-a** [comment]
The fix was released in version 1.9.0.

Verified using the following definition:
```json
{
    "Comment": "An example of the Amazon States Language using a choice state.",
    "StartAt": "FirstState",
    "States": {
        "FirstState": {
            "Type": "Choice",
            "Choices": [
                {
                    "Variable": "$.foo",
                    "NumericEquals": 1,
                    "Next": "Succeed"
                },
                {
                    "Variable": "$.foo",
                    "NumericEquals": 2,
                    "Next": "Fail"
                }
            ]
        },
        "Succeed": {
            "Type": "Succeed"
        },
        "Fail": {
            "Type": "Fail"
        }
    }
}
```
